JsonLd/Response.php

```diff
@@ -18,7 +18,7 @@
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
-class JsonLdResponse extends JsonResponse
+class Response extends JsonResponse
```

Comment:
Sure ? I think that `JsonLdResponse` won't introduce conflicts while `Response` will, the DX will be degraded if `use`s always have to be aliased.